docfx.json

```diff
@@ -95,7 +107,8 @@
     },
     "fileMetadata": {
       "ms.prod": {
-        "api/**.yml": ".net"
+        "api/**.yml": ".net",
+        "_csharplang/spec/*.md": ".net"
```

Comment:
you might wanna add ms.technology devlang-csharp as we do it in the c# topics, otherwise it goes to the blank bucket